### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by leoniechevalier**

### CODE REVIEW

**Comment by pwc** [comment]
LGTM ← “looks good to me”

File: src/skysim.py [hunk 1]
```diff
@@ -32,6 +33,19 @@ def generate_positions(nsrc):
         decs.append(dec + random.uniform(-1,1))
     return ras, decs
 
+def crop_to_circle(ras,decs,ra,dec,radius):
+    ras_circular=[]
+    decs_circular=[]
+
+    for i in range(len(ras)):
+        #checks if ra and dec fall inside the circle
+        if (ras[i]-ra)**2 + (decs[i]-dec)**2 < radius**2:
```
Comment (pwc) [inline src/skysim.py:42]: Worth using `pow` for this?

File: src/skysim.py [hunk 2]
```diff
@@ -3,17 +3,18 @@
...
 import math
 import random
 
 NSRC = 1_000
+ra = '01:42:44.3'
+dec = '40:16:09'
 
 def generate_positions(nsrc):
     # Determine Andromeda location in ra/dec degrees
     # from wikipedia
```
Comment (pwc) [inline src/skysim.py:16]: Is Wikipedia an OK source? And this comment is now missing its code...

File: mymodule/sky_sim.py
Comment (leoniechevalier) [inline mymodule/sky_sim.py:1]: i got the numbers mixed up here

File: src/skysim.py [hunk 3]
```diff
@@ -32,6 +33,19 @@ def generate_positions(nsrc):
         decs.append(dec + random.uniform(-1,1))
     return ras, decs
 
+def crop_to_circle(ras,decs,ra,dec,radius):
+    ras_circular=[]
+    decs_circular=[]
+
+    for i in range(len(ras)):
```
Comment (leoniechevalier) [inline src/skysim.py:40]: is this th best way to draw circles ?

File: src/skysim.py
Comment (leoniechevalier) [inline src/skysim.py:1]: blah blah

File: src/skysim.py [hunk 4]
```diff
@@ -3,17 +3,18 @@
 Script to generate a simulated star catalogue, for stars around the Andromeda
 galaxy
 """
-
+#new comment here
 import math
 import random
 
 NSRC = 1_000
```
Comment (leoniechevalier) [inline src/skysim.py:10]: blah blah blah

**Review by leoniechevalier — COMMENTED** [review]
what about these

**Pull request merged**